argo/workflow/titan-bank.yaml

```diff
@@ -62,7 +62,7 @@ spec:
                   value: "{{workflow.parameters.loki-username}}"
                 - name: loki-password
                   value: "{{workflow.parameters.loki-password}}"
-                - name: tikv-config
+                - name: tikv_config
```

Comment:
This should be `tikv-config`

Reply:
Yeah, It's confusing, but we mix the use of underscore and dash on argo template. I will unify them on tomorrow.

Reply:
Sounds good. You are going to update all argo template arguments to be unified with tipocket flag?

Reply:
If you don't mind I can do that today.

Reply:
ahh, great.